doc/semitrans.xml

```diff
@@ -12,18 +12,20 @@
 <ManSection>
   <Attr Name = "GeneratorsSmallest" Arg = "S" Label = "for a transformation
     semigroup"/>
-  <Returns>A generating set of transformations.</Returns>
+  <Returns>The smallest generating set of <A>S</A>.</Returns>
```

Comment:
The `<Returns>` statement should only state the *type* of object being returned (and if it is a list, the type of objects inside that list).

Accordingly, this one should just be "`<Returns>A set of transformations.</Returns>`".